rskj-core/src/main/java/co/rsk/peg/feeperkb/FeePerKbResponseCodes.java

```diff
@@ -0,0 +1,27 @@
+package co.rsk.peg.feeperkb;
+
+public enum FeePerKbResponseCodes {
+        FEE_PER_KB_GENERIC_ERROR(-10),
```

Comment:
The other two `return -10` appear in `addFederatorPublicKey` and `commitFederation` methods, which belong to the federation change process. We could have a `FederationChangeErrorCode` to handle these exceptions.
And this would make total sense when having the FederationChangeSupport. 

I.e., each support would have its specific error responses.

wdyt?

Reply:
Mmm, looking at the code in `Bridge` class, it seems that there are a bunch of places returning "parsing errors" related to the args input.

See:

```
    public Integer voteFeePerKbChange(Object[] args) {
        logger.trace("voteFeePerKbChange");

        Coin feePerKb;
        try {
            feePerKb = Coin.valueOf(((BigInteger) args[0]).longValueExact());
        } catch (Exception e) {
            logger.warn("Exception in voteFeePerKbChange", e);
            return -10;
        }
```

```
    public Integer commitFederation(Object[] args) throws BridgeIllegalArgumentException {
        logger.trace("commitFederation");

        byte[] hash;
        try {
            hash = (byte[]) args[0];
        } catch (Exception e) {
            logger.warn("Exception in commitFederation", e);
            return -10;
        }
```

```
    public Integer addFederatorPublicKey(Object[] args) throws BridgeIllegalArgumentException {
        logger.trace("addFederatorPublicKey");

        byte[] publicKeyBytes;
        try {
            publicKeyBytes = (byte[]) args[0];
        } catch (Exception e) {
            logger.warn("Exception in addFederatorPublicKey", e);
            return -10;
        }
```

```
    public long getLockWhitelistEntryByAddress(Object[] args) {
        logger.trace("getLockWhitelistEntryByAddress");

        String addressBase58;
        try {
            addressBase58 = (String) args[0];
        } catch (Exception e) {
            logger.warn("Exception in getLockWhitelistEntryByAddress", e);
            return LOCK_WHITELIST_INVALID_ADDRESS_FORMAT_ERROR_CODE;
        }
```

```
    public Integer addOneOffLockWhitelistAddress(Object[] args) {
        logger.trace("addOneOffLockWhitelistAddress");

        String addressBase58;
        BigInteger maxTransferValue;
        try {
            addressBase58 = (String) args[0];
            maxTransferValue = (BigInteger) args[1];
        } catch (Exception e) {
            logger.warn("Exception in addOneOffLockWhitelistAddress", e);
            return 0;
        }
```

Reply:
On one hand, I believe we should stick with the approach we're adopting for FeePerKbSupport, where each support would have its specific error code responses.

However, I also think we should introduce a new exception, `BridgeArgumentsException` (something like that), to handle these kinds of exceptions.
This way, we would handle exceptions differently across different layers of the code.